tests/rados/rados_prep.py

```diff
@@ -369,3 +497,43 @@ def create_pool(node: CephAdmin, pool_name: str, pg_num: int = 64, **kwargs) ->
 
     log.info(f"Created pool {pool_name} successfully")
     return True
+
+
+def run_ceph_command(node: CephAdmin, action: str) -> dict:
+    """
+    Runs ceph commands with json tag for the action specified otherwise treats action as command
+    and returns formatted output
+    Args:
+        node: Cephadm node where the commands need to be executed
+        action: The various commands that can be run are:
+            1. "ceph_report" -> cluster status
+            2. "daemon_info" -> info on all daemons running on cluster
+            3. "mgr_modules" -> list of modules available in mgr module
+            4. "cluster_usage" -> provides details about cluster/pool usage
+            5. "config_dump" -> Config changes made on cluster ( any daemon )
+            6. "cluster_usage_detail" -> detailed clustr usage info including limits set
+            7. "osd_details" -> info on OSD collected from Crush tree
+            8. "mon_details" -> monitor quorum and epoch details
+
+    Returns:
+    """
+
+    cmd_map = {
+        "ceph_report": r"ceph report",
+        "daemon_info": r"ceph orch ps",
+        "mgr_modules": r"ceph mgr module ls",
+        "cluster_usage": r"ceph df",
+        "config_dump": r"ceph config dump",
+        "cluster_usage_detail": r"ceph df detail",
+        "osd_details": r"ceph osd df tree",
+        "mon_details": r"ceph mon dump",
+    }
```

Comment:
IMO, allow the caller to provide the exact command instead of us having a map... it would allow first timer's have a better understanding. Also, you may want to generalize it for backward compatability

Reply:
That was my initial thought. But if some command implementation changes, having command map, we just need to change command in 1 place for commonly used commands. Hence created cmd_map.

Observed this for `ceph mon stat --format json-pretty ` is not working, and needed to change this to `ceph mon dump  --format json-pretty`

Having this would provide command reference.

These were my thoughts. Please let me know if I need to modify this.